history/characters/10000_orc.txt

```diff
@@ -610,8 +610,8 @@
 		employer = 10000
 		add_prestige = 750		# Should be able to invite clans when he's Warchief
 		add_piety = 375
-		add_claim = e_horde
-		add_claim = k_orc_blackrock
+		add_pressed_claim = e_horde
```

Comment:
Could you replace other `add_claim` with `add_pressed_claim` to fix all kinds of this bug?

Reply:
As requested, I replaced all references with `add_pressed_claim`: 0da8ab8dd2c90f253fe531bf6a5862ee03542889